pan-domain-node/.nvmrc

```diff
@@ -0,0 +1 @@
+6
```

Comment:
worth targeting a higher version of node?

Reply:
Yep definitely now Node 6 lambdas are EOL